README.md

```diff
@@ -4,63 +4,51 @@ Welcome to the home page for the MessageFormat Working Group, a subgroup of the
 
 ## Charter
 
-The MessageFormat Working Group (MFWG) is tasked with developing an industry standard
-for the representation of localizable message strings to be a successor to 
-[ICU MessageFormat](https://unicode-org.github.io/icu/userguide/format_parse/messages/). 
-MFWG will recommend how to remove redundancies, 
-make the syntax more usable, 
-and support more complex features, such as gender, inflections, and speech. 
-MFWG will also consider the integration of the new standard with programming environments, 
-including, but not limited to, ICU, DOM, and ECMAScript, and with localization platform interchange. 
-The output of MFWG will be a specification for the new syntax.
-
-- [Why ICU MessageFormat Needs a Successor](docs/why_mf_next.md)
-- [Goals and Non-Goals](docs/goals.md)
-
-## MessageFormat 2 Final Candidate
-
-The [MessageFormat 2 specification](./spec/) is a new part of
+The MessageFormat Working Group (MFWG) is tasked with developing and supporting an industry standard
+for the representation of localizable message strings. 
+MessageFormat is design to support software developers, translators, and end users with fluent messages
+and locally-adapted presentation of data values
+while developing support for increasingly complex features, such as gender, inflections, and speech. 
+Our goal is to provide an interoperable syntax and associated processing that is
+capable of being adopted by any presentation framework or programming environement.
+
+The working group is currently developing a charter for additional, post-release goals,
+including support for some deliverables not included in LDML47.
+A partial list might include:
+- XLIFF round trip mappings for MF2 messages
+- a machine-readable function description format
```

Comment:
I'd prefer leaving this out for now, and adding it once we do actually have new goals.

```suggestion
```

Reply:
I don't know how long to expect for the "chartering" phase. If it lasts awhile, might be good to note what we're doing. But I can easily be persuaded otherwise.

Reply:
I think we want to approach this as successive refinement, and not put in just ideas for what we might do, but rather wait until we have settled on what we want to do, eg for 48 (release in Oct).

Reply:
A good charter should run for years, not just a single release. We should list the goals of our charter here--when they are available. But I tend to agree that this placeholding is not necessary until then.